xml/net/server/protocol.xml

```diff
@@ -577,12 +655,14 @@
     </struct>
 
     <struct name="MapDrainDamageOther">
+        <comment>Another player taking damage from a map HP drain</comment>
         <field name="player_id" type="short"/>
         <field name="hp_percentage" type="char"/>
         <field name="damage" type="short"/>
     </struct>
 
     <struct name="GlobalBackfillMessage">
+        <comment>A global chat message</comment>
```

Comment:
Should probably say "Sent when the player re-opens the global chat tab" or something. I can't remember what the official behavior is for this.